readme.txt

```diff
@@ -39,7 +39,13 @@ for tips on styling your site to work nicely with UIO.
 = Are there any themes themes that already work well with UIO? =
 
 The plugin has been tested with several themes, and we found that the Miranda theme responds very nicely.
+The default Twenty Seventeen theme also works fairly well, although some layout tweaks may be required for some preference combinations.
 
 == Issue Tracker ==
 
-http://issues.fluidproject.org/browse/FLOE/component/11510#selectedTab=com.atlassian.jira.plugin.system.project%3Acomponent-issues-panel
\ No newline at end of file
+http://issues.fluidproject.org/browse/FLOE/component/11510#selectedTab=com.atlassian.jira.plugin.system.project%3Acomponent-issues-panel
```

Comment:
This link doesn't work. Also, the link at line 36 points at an UIO doc on the wiki.fluidproject.org site. Shall it be updated to point to the infusion doc site?